go/common/host/services.go

```diff
@@ -31,7 +23,7 @@ const (
 // They should be resilient to services availability, because the construction ordering is not guaranteed.
 type Service interface {
 	Start() error
-	Stop() error
+	Stop()
```

Comment:
In the past we had issues with tests because stop was not stopping correctly. We would either ignore the error or swallow it.

Prolly best to always check to for errors ?

Reply:
All we can do is log the error if it gets returned, I think the contract should be that the services try their best to stop any go-routines they own and log any errors in the process.